### ISSUE TITLE
Unexpected end of input

### ISSUE THREAD

**Issue by Musicpac** [schul-cloud/schulcloud-flutter#341]
After installing the app I was not able to find any functionality in it. I was able to login to the brandenburg schulcloud but non of the contents could be displayed. The pink screens are everywhere. The main error displayed is "FormatException: Unexpected end of input(at character 1). If you try to go to your profile the error displays a JsonParserfail.

![WhatsApp Image 2020-08-19 at 20 47 52 (1)](https://user-images.githubusercontent.com/33967941/90677275-67d3ef80-e25d-11ea-8e50-ccb0533afcbe.jpeg)
![WhatsApp Image 2020-08-19 at 20 47 52](https://user-images.githubusercontent.com/33967941/90677282-6b677680-e25d-11ea-8450-2f952b450588.jpeg)

**Comment by MarcelGarus** [comment]
Yes, this happened a lot. I think we fixed it in https://github.com/hpi-schul-cloud/schulcloud-flutter/pull/334 – and an even more comprehensive PR for error handling is in the works at https://github.com/hpi-schul-cloud/schulcloud-flutter/pull/337.
Either way, I might be wrong, so be sure to check if the error still persists with the version from https://github.com/hpi-schul-cloud/schulcloud-flutter/tree/handle-errors-better.

**Comment by Musicpac** [comment]
The error got kinda solved. now the problem changed to a ressources not found and the session being invalid. Does my account need to be validated?
![flutter](https://user-images.githubusercontent.com/33967941/90688228-6fe85b00-e26e-11ea-96e2-4e30290c071a.PNG)

**Comment by JonasWanke** [comment]
Currently, you are signed out automatically after two hours of inactivity. This is a limitation of the server, but will hopefully be changed soon. If you see such an error, you'll have to sign out via the banner at the top or via the account dialog, and then sign back in.

As a brief background on the `FormatException`s you reported initially: Many of the server's API routes are temporarily blocked. Hence we cannot currently access assignments or users, for example. This is a temporary server limitation as well, but the backend team is actively working on unlocking these routes. The PRs mentioned by @marcelgarus just change the appearance of these errors.
